charts/seed-bootstrap/charts/loki/templates/statefulset.yaml

```diff
@@ -42,6 +42,21 @@ spec:
         fsGroup: 10001
         fsGroupChangePolicy: "OnRootMismatch"
       priorityClassName: {{ .Values.priorityClassName }}
+      initContainers:
+      - command:
+        - bash
+        - -c
+        - tune2fs -O large_dir $(lsblk -o KNAME,MOUNTPOINTS | gawk 'NR>1 {if($2=="/data") {print "/dev/"$1}}')
+        image: {{ index .Values.global.images "tune2fs" }}
+        name: init-large-dir
+        securityContext:
+          privileged: true
```

Comment:
Do you really need this init container to be privileged? Can you elaborate what fails in non-privileged mode?

/hold

Reply:
In non-privileged mode it fails with:
```
tune2fs 1.46.6 (1-Feb-2023)
tune2fs: No such file or directory while trying to open /dev/nvme1n1
```

If you want, you can go over the linux capabilities and check if there is one allowing access to the devices under `/dev/`. If not I guess it is fine as anyways it is an init-container.

/unhold

Reply:
Fast search in [linux-capabilities](https://man7.org/linux/man-pages/man7/capabilities.7.html) for `dev` narrow the option to `CAP_SYS_ADMIN` and `CAP_SYS_RAWIO`.
```
  capabilities:
    add: ["CAP_SYS_ADMIN", "CAP_SYS_RAWIO"]
```
I got:
```
tune2fs 1.46.6 (1-Feb-2023)
tune2fs: No such file or directory while trying to open /dev/sdd
Couldn't find valid filesystem superblock.
```

Reply:
Okay, then let's keep it privileged.